src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -914,6 +906,14 @@ default User findByIdElseThrow(long userId) throws EntityNotFoundException {
             """)
     Set<User> getUsersWithAccessTokenNull();
 
+    @Query("""
+            SELECT COUNT(user) > 0
+            FROM User user
+            WHERE user.login = :login
+                AND :#{T(de.tum.in.www1.artemis.domain.Authority).ADMIN_AUTHORITY} MEMBER OF user.authorities
+            """)
```

Comment:
Depending on the user count performance could also be improved using EXISTS instead of COUNT